tests/autofix_files/noqa.py.diff

```diff
@@ -4,49 +4,49 @@
 
  # fmt: off
  async def foo_no_noqa():
--    with trio.fail_after(5):  # TRIO100: 9, 'trio', 'fail_after'
--        yield  # TRIO911: 8, "yield", Statement("function definition", lineno-2)
+-    with trio.fail_after(5):  # ASYNC100: 9, 'trio', 'fail_after'
+-        yield  # ASYNC911: 8, "yield", Statement("function definition", lineno-2)
 +    await trio.lowlevel.checkpoint()
-+    # TRIO100: 9, 'trio', 'fail_after'
-+    yield  # TRIO911: 8, "yield", Statement("function definition", lineno-2)
++    # ASYNC100: 9, 'trio', 'fail_after'
++    yield  # ASYNC911: 8, "yield", Statement("function definition", lineno-2)
      await trio.lowlevel.checkpoint()
```

Comment:
Unintended semantic changes in this file?  And a few others below.

Reply:
Oh, *very* good catch. Need to investigate!

Reply:
Reminder what the different files are for, from `CONTRIBUTING.md`:
> ### `# AUTOFIX`
> Files in `tests/eval_files` with this marker will have two files in `tests/autofix_files/`. One with the same name containing the code after being autofixed, and a diff file between those two.  
> During tests the result of running the checker on the eval file with autofix enabled will be compared to the content of the autofix file and will print a diff (if `-s` is on) and assert that the content is the same. `--generate-autofix` is added as a pytest flag to ease development, which will print a diff (with `-s`) and overwrite the content of the autofix file.  
> Files without this marker will be checked that they *don't* modify the file content.


Theree are no unintended changes in `tests/eval_files/noqa.py`, nor in `tests/autofix_files/noqa.py`, so there *shouldn't* be unintended changes in `tests/autofix_files/noqa.py.diff`. And when actually looking at https://github.com/python-trio/flake8-trio/blob/022fc3a4f661ec42afac5a21eb0404fe13f62380/tests/autofix_files/noqa.py.diff and comparing it to https://github.com/python-trio/flake8-trio/blob/780e5cd685e1f4ee187064567aca5890f4a7c0c4/tests/autofix_files/noqa.py.diff there in fact aren't any unintended changes.

Looking closer at the diff, the confusion stems from `.diff` files being syntax-highlighted, so we're viewing a diff of a `.diff` file. The solid green/red background, and the righmost +/-, is the diff from the file, and the line number having a green/red background & the leftmost +/- is the content of the diff file actually being changed in this commit. So e.g. line 9 is *not* changed in the commit, and there are no semantic changes.

The green on line 9 is to indicate that when running autofix on `eval_files/noqa.py` we have a checkpoint added.